tt_eager/tt_dnn/op_library/moreh_norm/moreh_norm_other/kernels/moreh_norm_other_kernel.cpp

```diff
@@ -51,53 +51,59 @@ void MAIN {
     for (uint32_t outer_idx = 0; outer_idx < num_output_tiles_per_core; ++outer_idx) {
         for (uint32_t inner_idx = 0; inner_idx < num_reduced_tiles_along_dim; ++inner_idx) {
             // |x|
-            ACQ();
+			tile_regs_acquire();
```

Comment:
The code alignment appears to be off. Could you kindly reformat it?

Reply:
I have applied formatting to the kernel code.